next/modules/navigation/MobileLatestEvents.tsx

```diff
@@ -10,11 +10,15 @@ const MobileLatestEvents = () => {
   if (!upcomingEvents?.data.length) {
     return null
   }
-
+  
   return (
     <ul className="mt-2 flex flex-col gap-y-3">
-      {upcomingEvents.data.filter(isDefined).map((event) => {
-        return <EventRow event={event} />
+      {upcomingEvents.data.filter(isDefined).map((event, index) => {
+        return (
+          <li key={event?.id ?? index}>
```

Comment:
Please do not mix it - use either `event.id` if it's unique and defined for all events, or only `index` if it's not.